drivers/driver-test/src/commonMain/kotlin/com/squareup/sqldelight/driver/test/DriverTest.kt

```diff
@@ -81,7 +82,9 @@ abstract class DriverTest {
     driver.close()
   }
 
-  @Test fun `insert can run multiple times`() {
+  @JsName("insertCanRunMultipleTimes")
```

Comment:
I would prefer we just rename the function name than maintain two names for every test.

Reply:
I agree. Any preference style? spaces are not supported on js, but kebab style is.

Reply:
None! Underscores, camel, kebab... I don't think it matters all that much.